common/djangoapps/enrollment/api.py

```diff
@@ -37,8 +37,9 @@ def get_enrollments(user_id):
                 "mode": "honor",
                 "is_active": True,
                 "user": "Bob",
-                "course": {
```

Comment:
Documentation here was incorrect. The key included in the response is `course_details`, not `course`.